opendbc/safety/safety/safety_hyundai_flags.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+// common flags
+extern bool hyundai_ev_gas_signal;
+bool hyundai_ev_gas_signal = false;
+
+extern bool hyundai_hybrid_gas_signal;
+bool hyundai_hybrid_gas_signal = false;
+
+extern bool hyundai_longitudinal;
+bool hyundai_longitudinal = false;
+
+extern bool hyundai_camera_scc;
+bool hyundai_camera_scc = false;
+
+extern bool hyundai_canfd_lka_steering;
+bool hyundai_canfd_lka_steering = false;
+
+extern bool hyundai_fcev_gas_signal;
+bool hyundai_fcev_gas_signal = false;
+
+// shared flags for non CAN FD cars
+extern bool hyundai_alt_limits;
+bool hyundai_alt_limits = false;
+
+extern bool hyundai_alt_limits_2;
+bool hyundai_alt_limits_2 = false;
+
+// shared flags for CAN FD cars
+extern bool hyundai_canfd_alt_buttons;
+bool hyundai_canfd_alt_buttons = false;
+
+extern bool hyundai_canfd_lka_steering_alt;
+bool hyundai_canfd_lka_steering_alt = false;
+
+void hyundai_common_flags(uint16_t param) {
+  const int HYUNDAI_PARAM_EV_GAS = 1;
+  const int HYUNDAI_PARAM_HYBRID_GAS = 2;
+  const int HYUNDAI_PARAM_CAMERA_SCC = 8;
+  const int HYUNDAI_PARAM_CANFD_LKA_STEERING = 16;  // TODO: rename for support with CAN/CAN FD Blended platforms
+  const int HYUNDAI_PARAM_FCEV_GAS = 256;
```

Comment:
This doesn't feel any cleaner and still has the same issue of out of order flags

Reply:
I can try using anonymous enum, similar to this:
https://github.com/commaai/opendbc/blob/75d4e0eb5c8fa961a665ec020eb816a50eed231c/opendbc/safety/safety/safety_hyundai_common.h#L13-L18